concepticondata/conceptlists/Tolmie-1884-211.tsv

```diff
@@ -0,0 +1,212 @@
+Tolmie-1884-211-8	8	My mother (said by son)		
+Tolmie-1884-211-9	9	My mother (said by daughter)		
+Tolmie-1884-211-10	10	My husband		
+Tolmie-1884-211-11	11	My wife		
+Tolmie-1884-211-12	12	My son (said by father)		
+Tolmie-1884-211-13	13	My son (said by mother)		
+Tolmie-1884-211-14	14	My daughter (said by father)		
+Tolmie-1884-211-15	15	My daughter (said by mother)		
+Tolmie-1884-211-16	16	My elder brother		
+Tolmie-1884-211-17	17	My younger brother		
+Tolmie-1884-211-18	18	My elder sister		
+Tolmie-1884-211-19	19	My younger sister		
+Tolmie-1884-211-20	20	An Indian		
+Tolmie-1884-211-21	21	People	789	PEOPLE
+Tolmie-1884-211-22	22	Head	1256	HEAD
+Tolmie-1884-211-23	23	Hair	1040	HAIR
+Tolmie-1884-211-24	24	Face	1560	FACE
+Tolmie-1884-211-25	25	Forehead	123	FOREHEAD
+Tolmie-1884-211-26	26	Ear	1247	EAR
+Tolmie-1884-211-27	27	Eye	1248	EYE
+Tolmie-1884-211-28	28	Nose	1221	NOSE
+Tolmie-1884-211-29	29	Mouth	674	MOUTH
+Tolmie-1884-211-30	30	Tongue	1205	TONGUE
+Tolmie-1884-211-31	31	Teeth	1380	TOOTH
+Tolmie-1884-211-32	32	Beard	678	BEARD
+Tolmie-1884-211-33	33	Neck	1333	NECK
+Tolmie-1884-211-34	34	Arm	1673	ARM
+Tolmie-1884-211-35	35	Hand	1277	HAND
+Tolmie-1884-211-36	36	Fingers	1303	FINGER
+Tolmie-1884-211-37	37	Thumb	1781	THUMB
+Tolmie-1884-211-38	38	Nails	1896	FINGERNAIL OR TOENAIL
+Tolmie-1884-211-39	39	Body	1480	BODY
+Tolmie-1884-211-40	40	Chest	1592	CHEST
+Tolmie-1884-211-41	41	Belly	1251	BELLY
+Tolmie-1884-211-42	42	Female breasts	1402	BREAST
+Tolmie-1884-211-43	43	Leg	1297	LEG
+Tolmie-1884-211-44	44	Foot	1301	FOOT
+Tolmie-1884-211-45	45	Toes	1389	TOE
+Tolmie-1884-211-46	46	Bone	1394	BONE
+Tolmie-1884-211-47	47	Heart	1223	HEART
+Tolmie-1884-211-48	48	Blood	946	BLOOD
+Tolmie-1884-211-49	49	Town, village		
+Tolmie-1884-211-50	50	Chief	2394	CHIEF
+Tolmie-1884-211-51	51	Warrior		
+Tolmie-1884-211-52	52	Friend	1325	FRIEND
+Tolmie-1884-211-53	53	House	1252	HOUSE
+Tolmie-1884-211-54	54	Skin lodge		
+Tolmie-1884-211-55	55	Kettle	854	KETTLE
+Tolmie-1884-211-56	56	Bow	994	BOW
+Tolmie-1884-211-57	57	Arrow	977	ARROW
+Tolmie-1884-211-58	58	Axe, hatchet	2083	AXE (CLASSIFIER)
+Tolmie-1884-211-59	59	Knife	1352	KNIFE
+Tolmie-1884-211-60	60	Canoe	1970	CANOE
+Tolmie-1884-211-61	61	Moccasins		
+Tolmie-1884-211-62	62	Pipe	1124	PIPE
+Tolmie-1884-211-63	63	Tobacco	974	TOBACCO
+Tolmie-1884-211-64	64	Sky	1732	SKY
+Tolmie-1884-211-65	65	Sun	1343	SUN
+Tolmie-1884-211-66	66	Moon	1313	MOON
+Tolmie-1884-211-67	67	Star	1430	STAR
+Tolmie-1884-211-68	68	Day	1225	DAY (NOT NIGHT)
+Tolmie-1884-211-69	69	Night	1233	NIGHT
+Tolmie-1884-211-70	70	Morning	1339	MORNING
+Tolmie-1884-211-71	71	Evening	1629	EVENING
+Tolmie-1884-211-72	72	Spring	1537	SPRINGTIME
+Tolmie-1884-211-73	73	Summer	1342	SUMMER
+Tolmie-1884-211-74	74	Autumn	1276	AUTUMN
+Tolmie-1884-211-75	75	Winter	1471	WINTER
+Tolmie-1884-211-76	76	Wind	960	WIND
+Tolmie-1884-211-77	77	Thunder	1150	THUNDER
+Tolmie-1884-211-78	78	Lightning	1654	LIGHTNING
+Tolmie-1884-211-79	79	Rain	658	RAIN (PRECIPITATION)
```

Comment:
Using the noun here is justified by the context, an interesting example of how linking depends on structure of a concept list.